### ISSUE TITLE
 Disable ANSI control characters in output

### ISSUE THREAD

**Issue by andrus** [dflib/jjava#24]
There was a recent issue in iJava - https://github.com/SpencerPark/IJava/issues/163 . Let's check if we can fix it on our end.

**Comment by emcastro** [comment]
Use of ANSI control characters is annoying when using VSCode Jupyter in standard dark mode. Some error messages are written black on dark gray.
![image](https://github.com/user-attachments/assets/bebfe1c1-b8e6-4b6e-af53-9901e0443152)
Try to read the message `|     null.toString()`!

**Comment by andrus** [comment]
Thanks for providing an example. We'll need to look into it

**Comment by stariy95** [comment]
A quick note, in Bootique CLI we are using https://github.com/fusesource/jansi library to work with ASCII control codes.

**Comment by andrus** [comment]
I'd rather we don't add any third-party dependencies. But I am not fully up to speed on the complexity involved.

**Comment by stariy95** [comment]
Yes, adding this library could be excessive (and it uses native binaries). 
We may only look at how they solved the detection part, and use it to just turn off formatting on terminals we are not sure it is safe to use.
